CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # [Versions](https://mui.com/versions/)
 
+## v6.1.8
+
+<!-- generated comparing v6.1.7..master -->
+
+_Nov 20, 2024_
+
+A big thanks to the 9 contributors who made this release possible.
+
+TODO INSERT HIGHLIGHTS
+
+### `@mui/material@6.1.8`
+
+- [material-ui][Autocomplete] Use `ul` element for the listbox (#44422) @DiegoAndai
+- [Grid2] Remove item and zeroMinWidth classes from grid2Classes (#44419) @sai6855
+- [materia-ui][StepIcon] Add SvgIconOwnProps type to StepIcon props (#44337) @sai6855
+- [material-ui] Add generic back to `useMediaQuery` to prevent a breaking change (#44455) @siriwatknp
```

Comment:
```suggestion
- [StepIcon] Add SvgIconOwnProps type to StepIcon props (#44337) @sai6855
- Add generic back to `useMediaQuery` to prevent a breaking change (#44455) @siriwatknp
```